sdk/tests/client/node_api/core.rs

```diff
@@ -357,3 +357,27 @@ async fn test_call_plugin_route() {
     // Just check name as info can change between 2 calls
     assert_eq!(plugin_res.node_info.name, info.node_info.name);
 }
+
+#[ignore]
+#[tokio::test]
+async fn test_get_routes() {
+    let client = setup_client_with_node_health_ignored().await;
+
+    let routes_response = client.get_routes().await.unwrap();
+    assert!(!routes_response.routes.is_empty());
```

Comment:
Probably better assertions than this

Reply:
could you give an example? This returns only the list of the routes. The list is dynamic because depends on what nodes are part of the tangle.

Reply:
Is there a minimum requirement maybe?

Reply:
I find there is a only one `core/v2`, which is available by default. I've added the validation for it. The rest is available through plugins

Reply:
good enough, thanks